Rewrite anonym function "syncTree" for compiler and debugging reasons.

MarkusBordihn opened the pull request:
See #448 for more details.

rachel-fenichel commented on core/toolbox.js:
```diff
@@ -265,92 +265,99 @@ Blockly.Toolbox.prototype.position = function() {
+/**
+ * Sync trees of the toolbox.
+ * @param {Node} treeIn DOM tree of blocks, or null.
+ * @param {Blockly.Toolbox.TreeControl} treeOut
```
Now that this isn't anonymous you should be able to get rid of the treeOut parameter and just used this.tree_ in the function.

MarkusBordihn replied:
I have leave this, in the case that its needed later.

rachel-fenichel commented:
This diff is a bit hard to read--can you summarize the change you made?

It looks like you've moved syncTree to be defined on the toolbox prototype.  Anything else?

MarkusBordihn commented:
Yes, in general I have moved syncTree to an prototype and adjust the parameters and removed unneeded var declarations. I have not changed the logic of syncTree itself.

rachel-fenichel commented on core/toolbox.js:
```diff
@@ -265,92 +265,99 @@ Blockly.Toolbox.prototype.position = function() {
+  Blockly.resizeSvgContents(this.workspace_);
+};
+
+
```
One newline between functions, here and below.

rachel-fenichel commented:
One comment then LGTM.

MarkusBordihn commented:
Done.
